ocrd/ocrd/task_sequence.py

```diff
@@ -74,16 +83,18 @@ def __str__(self):
             ret += ' -p %s' % self.parameter_path
         return ret
 
-def validate_tasks(tasks, workspace):
+def validate_tasks(tasks, workspace, page_id=None, overwrite=False):
     report = ValidationReport()
     prev_output_file_grps = workspace.mets.file_groups
 
+    first_task = tasks.pop(0)
+    first_task.validate()
+
     # first task: check input/output file groups from METS
-    # TODO disable output_file_grps checks once CLI parameter 'overwrite' is implemented
-    WorkspaceValidator.check_file_grp(workspace, tasks[0].input_file_grps, tasks[0].output_file_grps, report)
+    WorkspaceValidator.check_file_grp(workspace, first_task.input_file_grps, first_task.output_file_grps, page_id, report)
 
-    prev_output_file_grps += tasks[0].output_file_grps
-    for task in tasks[1:]:
+    prev_output_file_grps += first_task.output_file_grps
+    for task in tasks:
         task.validate()
         # check either existing fileGrp or output-file group of previous task matches current input_file_group
         for input_file_grp in task.input_file_grps:
```

Comment:
Cannot comment/suggest on non-context lines, but actually my focus is on L92 (now L103):
How about an extra check that output fileGrps do not exist yet (at least for relevant pages):
```python
        if not overwrite:
            WorkspaceValidator.check_file_grp(workspace, '', task.output_file_grps, page_id, report)
```

Reply:
Makes sense, thx. 275a0a66